lib/hexpm_web/router.ex

```diff
@@ -118,6 +124,21 @@ defmodule HexpmWeb.Router do
     get "/password", PasswordController, :index, as: :dashboard_password
     post "/password", PasswordController, :update, as: :dashboard_password
 
+    get "/security", SecurityController, :index, as: :dashboard_security
+    post "/security", SecurityController, :update, as: :dashboard_security
+
+    post "/security/rotate_recovery_codes", SecurityController, :rotate_recovery_codes,
+      as: :dashboard_security
+
+    get "/two_factor_auth/setup", TwoFactorAuthSetupController, :index,
+      as: :dashboard_two_factor_auth_setup
+
+    post "/two_factor_auth/setup", TwoFactorAuthSetupController, :update,
+      as: :dashboard_two_factor_auth_setup
+
+    post "/security/reset_auth_app", SecurityController, :reset_auth_app,
+      as: :dashboard_security_reset
```

Comment:
We don't have to prefix the action names with `dashboard_` since we are already in the context of the dashboard namespace.